docs/faq.md

```diff
@@ -48,25 +48,25 @@ add an explicit signal handler for `SIGTERM`.
 
 * Set the `stop_signal` to a signal which the application knows how to handle:
 
-      web:
-        build: .
-        stop_signal: SIGINT
+        web:
+          build: .
+          stop_signal: SIGINT
 
 * If you can't modify the application, wrap the application in a lightweight init
 system (like [s6](http://skarnet.org/software/s6/)) or a signal proxy (like
 [dumb-init](https://github.com/Yelp/dumb-init) or
 [tini](https://github.com/krallin/tini)).  Either of these wrappers take care of
 handling `SIGTERM` properly.
 
-## How do I run multiple copies of a Compose file on the same host?
+### How do I run multiple copies of a Compose file on the same host?
 
 Compose uses the project name to create unique identifiers for all of a
 project's  containers and other resources. To run multiple copies of a project,
 set a custom project name using the [`-p` command line
 option](./reference/overview.md) or the [`COMPOSE_PROJECT_NAME`
 environment variable](./reference/envvars.md#compose-project-name).
 
-## What's the difference between `up`, `run`, and `start`?
+### What's the difference between 'up', 'run', and 'start'?
```

Comment:
is this change also really a rendering issue? perhaps the rendering of code in a heading should be less dramatic?

idk, just asking - we should gather these small css problems and see if we can fix them consistently